components/Prize/PrizeExamples.jsx

```diff
@@ -0,0 +1,63 @@
+'use strict'
+
+import React from 'react'
+import PrizeItems from './PrizeItems'
+
+/** Temporarly div Style can be replaced by tool tip componenet **/
+const divStyle = {
```

Comment:
Yes. Basically, any css should be in css files. For the actual component, there would be one css file and for the component examples, you can have another css file.